Packs/Nmap/Integrations/Nmap/README.md

```diff
@@ -46,6 +46,9 @@ Scan targets with the given parameters
 | NMAP.Scan.Hosts.Services.State | unknown | The state of the service. | 
 | NMAP.Scan.Hosts.Services.Banner | unknown | Any captured banner from the service. | 
 | NMAP.Scan.Hosts.Services.Service | unknown | The service name. | 
+| NMAP.Scan.Hosts.ScriptResults.ID | unknown | The name of the script used. |
```

Comment:
let's mention the support in the new scripts, what are their names and what other relevant information needed to learn how to execute them (also in the readme)